busybox ps doesn't have an option to only select a specific pid.

--- a/tests/ppid-p.tst
+++ b/tests/ppid-p.tst
@@ -4,7 +4,7 @@ posix="true"
 
 test_OE -e 0 'PPID is parent process ID'
 echo $PPID >variable.out
-echo $(ps -o ppid= $$) >ps.out
+echo $(ps -o pid=,ppid= | awk -v pid=$$ '$1 == pid {print $2}') >ps.out
 diff variable.out ps.out
 __IN__
 
